IntegrationWithPharmacies/TenderApi/Model/Tender.cs

```diff
@@ -1,24 +1,23 @@
-﻿using System;
-using HealthClinic.CL.Model.Patient;
-
-namespace HealthClinic.CL.Model.Orders
-{
-    public class Tender: Entity
-    {
-        public DateTime ActiveUntil { get; set; }
-        public bool Closed{ get; set; }
-
-        public Tender() : base() { }
-
-        public Tender(int id, DateTime date, bool closed) : base(id)
-        {
-            ActiveUntil = date;
-            Closed = closed;
-        }
-        public Tender(DateTime date, bool closed)
-        {
-            ActiveUntil = date;
-            Closed = closed;
-        }
-    }
-}
+﻿using System;
+
+namespace TenderApi.Model
+{
+    public class Tender : Entity
+    {
+        public DateTime ActiveUntil { get; set; }
+        public bool Closed { get; set; }
+
+        public Tender() : base() { }
+
+        public Tender(int id, DateTime date, bool closed) : base(id)
+        {
+            ActiveUntil = date;
+            Closed = closed;
+        }
+        public Tender(DateTime date, bool closed)
+        {
+            ActiveUntil = date;
```

Comment:
Change parameter name to match left side for example it should be "ActiveUntil= activeUntil".
You could also try changing name of "ActiveUntil" into "ExpirationDate" if it makes sense in the given context.